api/cl_khr_command_buffer.asciidoc

```diff
@@ -126,11 +126,16 @@ from the sync-point values returned is implementation defined.
 
 ==== Simultaneous Use
 
-The optional simultaneous use capability was added to the extension so that
-vendors can support pipelined workflows, where command-buffers are repeatedly
-enqueued without blocking in user code. However, simultaneous use may result in
-command-buffers being more expensive to enqueue than in a sequential model, so
-the capability is optional to enable optimizations on command-buffer recording.
+The optional <<simultaneous-use, simultaneous use>> capability was added to the
+extension so that vendors could support concurrent execution of the same
+command-buffer. However, simultaneous use may result in command-buffers having
+a larger overhead to implement, so the capability is optional to enable
+optimizations when this usage isn't required by a user.
+
+Instead the goal for the base level of functionality provided by the extension
+is to support pipelined workflows. Where a command-buffer is repeatedly
+enqueued, with each enqueue expressing dependencies on any previous submissions,
+without the enqueue calls blocking in user code.
```

Comment:
Nitpick: I found this paragraph difficult to parse and I'd consider removing or rephrasing it.

If we decide to keep it, what is it trying to say that isn't covered in the first paragraph?

Reply:
I've just gone and removed it